test/routes/viewModels/taskLists/tasks/finalChecks.test.js

```diff
@@ -93,17 +100,24 @@ describe('final checks task', () => {
     test('should return multiple warning messages when multiple checks failed', () => {
```

Comment:
Do a test for segregation = no ?

Reply:
Adapted tests to check for this in latest commit 👍